frontend/src/util/axios.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const instance = axios.create({
+  baseURL: process.env.REACT_APP_SERVER_URL,
```

Comment:
why don't you simply do `process.env.NODE_ENV == 'production' ?  localhost:3001 : productionURL`?
We can get the production url from the current url using `window.path.split()`

Reply:
I run another server that takes up port 3001, so I actually had to run it on a different port and switch the frontend proxy. I'm not too sure about further hardcoding the server url.